app/assets/stylesheets/vendor/SyntaxHighlighter/SyntaxHighlighter.scss

```diff
@@ -1,5 +1,4 @@
-@import 'mixins';
-@import 'constants';
+@import '../../common/constants';
```

Comment:
@david-yz-liu I think I'll just look through the Hound config for CSS and make a separate PR with the changes.